tests/python_tests/common.py

```diff
@@ -474,3 +474,22 @@ def get_image_by_link(link):
         image = image.convert('RGB')
     image_data = np.array((np.array(image.getdata()) - 128).astype(np.byte)).reshape(1, 3, image.size[1], image.size[0])
     return Tensor(image_data)
+
+def get_streamer_with_results():
```

Comment:
as we agreed - we need to add to `run_llm_pipeline_with_ref` and `run_cb_pipeline_with_ref` and as they cover more cases (like streaming when stop strings are enabled)

Reply:
Extented `run_llm_pipeline` to use with streamer and reused `compare_generation_results`

Reply:
> to use with streamer

they don't use streamer now as default argument is None